contribs/gmf/src/services/themesservice.js

```diff
@@ -72,6 +76,21 @@ goog.inherits(gmf.Themes, goog.events.EventTarget);
 
 
 /**
+ * Init the service.
+ * @param {string} treeUrl URL to "themes" web service.
+ * @param {string} isThemePrivateUrl URL to check if theme is public.
+ * @export
+ */
+gmf.Themes.prototype.init = function(treeUrl, isThemePrivateUrl) {
```

Comment:
I've changed how to initialize the service (I use an init function to be coherent with others services) but that causes trouble with the Luxembourg project... Must I revert this change ?